### DISCUSSION TITLE
压缩插件不生效问题

### DISCUSSION THREAD

**Discussion by kiddmewhat** [apache/apisix#8092] [Q&A]
rpm方式安装了，
apisix-2.99.0-0.el7.x86_64.rpm
apisix-base-1.21.4.1.2-0.el7.x86_64.rpm

开启zip压缩功能，返回内容怎么没有压缩？
head头如下。插件应该是生效了

Content-Type →application/json;charset=utf-8
Transfer-Encoding →chunked
Connection →keep-alive
Date →Fri, 14 Oct 2022 12:02:23 GMT
Server →APISIX/2.99.0
Vary →Accept-Encoding
Content-Encoding →gzip

**Comment by tzssangglass** [comment]
`curl http://127.0.0.1:9080/index.html -i -H "Accept-Encoding: gzip"` do you add `H "Accept-Encoding: gzip"` when you send request?
